lib/arke_server/router.ex

```diff
@@ -133,6 +139,11 @@ defmodule ArkeServer.Router do
       delete("/unit/:unit_id", ProjectController, :delete)
     end
 
+    scope "/backoffice" do
```

Comment:
Can we find another key rather than `backoffice` for that? This scope will be useful for system/core related features